apps/scan/frontend/package.json

```diff
@@ -8,8 +8,11 @@
     "prodserver"
   ],
   "scripts": {
-    "build": "pnpm type-check && vite build",
+    "build": "pnpm --filter $npm_package_name... build:self",
```

Comment:
With these changes in place, can we now remove some of the extra build calls we have in our app frontend Makefile `make build` commands (the ones used by production build process)?

https://github.com/votingworks/vxsuite/blob/4ed2678acce4bce959d0982a0aab60a35927cc14/apps/scan/frontend/Makefile#L9-L11

Reply:
Good catch. Updated in b29ae0842.